### PR TITLE
App Router Changes

### PR DESCRIPTION
**Pull request by uellenberg**
This should fix some of the issues with the app router migration.

### CODE REVIEW

File: style/index.css [hunk 1]
```diff
@@ -1 +1 @@
-@layer reset, base, tokens, recipes, utilities;
```
Comment (TheDarkThief) [inline style/index.css:1]: What was changed here???

File: app/(mainSite)/impactathon/page.tsx
Comment (TheDarkThief) [inline app/(mainSite)/impactathon/page.tsx:1]: I believe we have spoken (I might be miss remembering) about this but it was noted that the Impactathon really is separate from the main site and I have spoken with the event's manager and he said he would prefer it does _not_ have the CSClub's header and footer.

**Review by TheDarkThief — CHANGES_REQUESTED** [review]
Thank you for fixing the styling issues, adding back the drawer, and reintroducing SEO.

However there are a few areas I believe could be improved upon and shouldn't take much time.
- as stated in a comment the event organizer would appreciate the CSClubs header being removed, and I agree as it really isn't part of the CSClub, I can undo that if you'd like.
- Next.JS has a system to do metadata based on path, it shouldn't be to hard to implement and would save us from having to define title every time, if you can't get this one done that's fine.
- Please provide a standardized Prettier config or a guide because I don't enjoy making sure two lines of the same code are actually the same besides using double quotes or single quotes

File: app/(mainSite)/minecraft/PageBody.tsx
Comment (TheDarkThief) [inline app/(mainSite)/minecraft/PageBody.tsx:1]: This page has had it's styling broken:
image

File: panda.config.ts [hunk 2]
```diff
@@ -1,74 +1,86 @@
...
+    defineGlobalStyles,
+    defineKeyframes,
+} from "@pandacss/dev";
 
 export const keyframes = defineKeyframes({
-  fadeInFromAboveAnim: {
-      '0%': { opacity: 0, transform: "translate(-50%, -80%)" },
-      '100%': { opacity: 1, transform: "translate(-50%, -50%)" },
-  },
+    fadeInFromAboveAnim: {
```
Comment (TheDarkThief) [inline panda.config.ts:8]: why?

File: panda.config.ts [hunk 3]
```diff
@@ -1,74 +1,86 @@
-import { defineConfig, defineGlobalStyles, defineKeyframes } from "@pandacss/dev";
+import {
```
Comment (TheDarkThief) [inline panda.config.ts:1]: why as well?

Comment (uellenberg) [reply panda.config.ts:1]: This is prettier.

File: panda.config.ts (hunk 2 above)
Comment (uellenberg) [reply panda.config.ts:8]: This is prettier.

File: app/(mainSite)/minecraft/PageBody.tsx
Comment (uellenberg) [reply app/(mainSite)/minecraft/PageBody.tsx:1]: I don't see any broken styling? Should the text be moved down more?

File: style/index.css (hunk 1 above)
Comment (uellenberg) [reply style/index.css:1]: This is prettier.

File: app/(mainSite)/impactathon/page.tsx
Comment (uellenberg) [reply app/(mainSite)/impactathon/page.tsx:1]: When I talked to Riko about this (for this very reason) he said he had no problem associating the Impactathon with the CS Club. I also haven't heard anything about this. I have no problem with removing the header and footer if that's what he wants - it's for his event, after all - but that was never communicated to me.

**Comment by uellenberg** [comment]
> Thank you for fixing the styling issues, adding back the drawer, and reintroducing SEO.
> 
> However there are a few areas I believe could be improved upon and shouldn't take much time.
> 
> * as stated in a comment the event organizer would appreciate the CSClubs header being removed, and I agree as it really isn't part of the CSClub, I can undo that if you'd like.
> * Next.JS has a system to do metadata based on path, it shouldn't be to hard to implement and would save us from having to define title every time, if you can't get this one done that's fine.
> * Please provide a standardized Prettier config or a guide because I don't enjoy making sure two lines of the same code are actually the same besides using double quotes or single quotes

I'm not really sure how path-based metadata could work, especially in the case of the "GameDev" page. We could implement something to do it, but I suspect it'd be more complicated than what we already have and would have to deal with edge cases like this anyway. Plus, some pages have extra details (like descriptions or images), so it's nicer to have it all in one spot. I think that the current approach (which requires one or two lines per file) is the simplest approach. 

There is, and always has been configs for prettier and eslint. You can run `prettier --write *` and `eslint --fix .` in order to apply them. It will used the style specified in the configuration automatically. You probably need to configure your editor to run prettier on save. 

I responded to the last point in one of the review comments.

File: app/(mainSite)/minecraft/PageBody.tsx
Comment (TheDarkThief) [reply app/(mainSite)/minecraft/PageBody.tsx:1]: This it is supposed to look like:
image

**Comment by TheDarkThief** [comment]
Perfect thank you

**Pull request merged**